Article/Article.md

```diff
@@ -17,89 +17,41 @@ abstract: |
 
 **Keywords:** replication study, gender differences, economic preferences
 
-
 # 1. Introduction
 
-Gender differences are nowadays extensively used as arguments and counter-arguments for decision and policy making, and the differences concerning the economic behaviors, such as happiness [@SPSU], competition [@CG; @GLL], or work preferences [@BG], are being studied in many sectors of the economy and economy-related fields.
+Gender differences concerning the economic behaviors, such as happiness [@SPSU], competition [@CG; @GLL], or work preferences [@BG], are being studied in sociology and economics. 
 
-One of the problems common for many experiments in social sciences is the lack of large and heterogeneous datasets that can be used to check for such differences reducing some of the bias induced, for example, by having students or specific sets of people interviewed for the study.
```

Comment:
Why don't you want to keep this? It seems to me a pretty natural connection to the next sentences, that are otherwise too "by their own"